tmt/base.py

```diff
@@ -324,7 +324,7 @@ class DependencyFmfId(FmfId):
     several extra keys.
     """
 
-    VALID_KEYS: ClassVar[List[str]] = FmfId.VALID_KEYS + ['destination', 'nick', 'type']
+    VALID_KEYS: ClassVar[List[str]] = [*FmfId.VALID_KEYS, 'destination', 'nick', 'type']
```

Comment:
What's the point of these changes? Is this supposed to me more readable? I'd say the original is a bit more clear.

Reply:
Ok, speed up. Makes sense.